script/openqa-clone-job

```diff
@@ -133,6 +133,10 @@ are cloned. Use C<0> to denote infinity.
 A shortcut for C<--skip-download --from HOST --host HOST> to clone a job within
 a local or remote instance.
 
+=item B<--repeat> N
+
+Do the same clone operation N times.
```

Comment:
I had been thinking about the name and the default. I think a more common and better name would be "count". For "repeat" with default "1" one might assume that a job is cloned *and* repeated one time equaling two jobs. For "count=1" the case is clear. Also I tink multiple other commands use a command line option with "-c/--count" so how about renaming? The logic can stay the same

Reply:
yeah, naming things + off by one in the same topic :) 
I get your point, this is already a 'clone' operation so if you want to do it only once you just avoid to use repeat. The documentation seems clear, do it N times when N by default is 1 . Should we give an error if the user specifies `repeat=1` ? 

Still, 'count' to me sounds more like a verb, like "please count and tell me how many jobs I cloned" . How about `times` ?

Reply:
hm, no, `times` is too confusing with either `time` or as in multiplying. I wanted to look up examples where other commands do something similar. I only found `dd` which says

```
  count=N         copy only N input blocks
```

If you don't like count then we can just stick with `repeat`

Reply:
> I had been thinking about the name and the default. I think a more common and better name would be "count". For "repeat" with default "1" one might assume that a job is cloned _and_ repeated one time equaling two jobs. For "count=1" the case is clear. Also I tink multiple other commands use a command line option with "-c/--count" so how about renaming? The logic can stay the same

looking at the documented example, 

```
openqa-clone-job --skip-chained-deps --repeat=50 --within-instance \
https://openqa.opensuse.org 123456  BUILD=poo32242_investigation \
_GROUP="Test Development:openSUSE Tumbleweed"
```

I read: I want to clone this job, repeat the operation 50 times and group it onto a build X, inside group Y.

@ilmanzo can you add a "This can be useful for Statistical investigation.."? or a similar comment, to supplement line 138:

> Clone job A, repeat the operation N times and group it onto a build X, inside group Y.
> 

And add an example command line.